### DISCUSSION TITLE
is this project for linux???

### DISCUSSION THREAD

**Discussion by aayushkakkar26** [practice404/clothes-virtual-try-on#21] [General]
Is this project is or linux system I have windows can I run this ? and you used somewhere c/usr/local so I have to change it right?

**Comment by SwayamInSync** [comment]
It requires at least 13GB GPU, so for general and easy to use purpose, setup according to colab environment.
But it can be tweaked easily for Windows system, just look for the setting up cells in google colab

**Comment by FCB-Sunny** [comment]
Thanks for your effort but I have some question. when I follow the all step in gradio jupyter notebook and click the submit, there is any actions.
And also, when I test the this repository, I run the run.py file but it require us the /workspace/inputs/test/openpose-img/model_1_rendered.png. How can I do? when I test this repository, can I make the model_1_rendered.png file?
The gradio UI can only upload the cloth image and model image.
I hope to hear your advice
thanks

**Comment by Pratush12** [comment]
How can we set it up in local? Can someone help here on the steps please?

**Comment by Souravtkrishna** [comment]
seg_final.pth

gmm_final.pth

alias_final.pth  where can i find these files
